src/state/util.ts

```diff
@@ -1,3 +1,4 @@
+import {AnyAction} from "redux";
```

Comment:
Tiny stylistic nit: should have spaces padding curlies in the `{ AnyAction }` import, and because we're only pulling one thing out of `./types`, can get away with onelining it.